src/LHScriptX/FeatureScriptCommands.h

```diff
@@ -21,111 +25,117 @@ class FeatureScriptCommands
-	static void CreateNewTownField(const ScriptCommandContext& ctx);
-	static void CreateSpellDispenser(const ScriptCommandContext& ctx);
-	static void LoadComputerPlayerPersonallty(const ScriptCommandContext& ctx);
-	static void SetComputerPlayerPersonallty(const ScriptCommandContext& ctx);
-	static void SetGlobalLandBalance(const ScriptCommandContext& ctx);
-	static void SetLandBalance(const ScriptCommandContext& ctx);
-	static void CreateDrinkWaypoint(const ScriptCommandContext& ctx);
-	static void SetTownInfluenceMultiplier(const ScriptCommandContext& ctx);
-	static void SetPlayerInfluenceMultiplier(const ScriptCommandContext& ctx);
-	static void SetTownBalanceBeliefScale(const ScriptCommandContext& ctx);
-	static void StartGameMessage(const ScriptCommandContext& ctx);
-	static void AddGameMessageLine(const ScriptCommandContext& ctx);
-	static void EditLevel(const ScriptCommandContext& ctx);
-	static void SetNighttime(const ScriptCommandContext& ctx);
-	static void MakeLastObjectArtifact(const ScriptCommandContext& ctx);
-	static void SetLostTownScale(const ScriptCommandContext& ctx);
+	static void CreateMist(glm::vec3 position, float param_2, int32_t param_3, float param_4, float param_5);
+	static void CreatePath(int32_t param_1, int32_t param_2, int32_t param_3, int32_t param_4);
+	static void CreateTown(int32_t townId, glm::vec3 position, const std::string& playerOwner, int32_t notUsed,
+	                       const std::string& civilisation);
+	static void SetTownBelief(int32_t townId, const std::string& playerOwner, float belief);
+	static void SetTownBeliefCap(int32_t townId, const std::string& playerOwner, float belief);
+	static void SetTownUninhabitable(int32_t townId);
+	static void SetTownCongregationPos(int32_t townId, glm::vec3 position);
+	static void CreateAbode(int32_t townId, glm::vec3 position, const std::string& abodeInfo, int32_t rotation, int32_t size,
+	                        int32_t foodAmount, int32_t woodAmount);
+	static void CreatePlannedAbode(int32_t townId, glm::vec3 position, const std::string& abodeInfo, int32_t rotation,
+	                               int32_t size, int32_t foodAmount, int32_t woodAmount);
+	static void CreateTownCentre(int32_t townId, glm::vec3 position, const std::string& abodeInfo, int32_t rotation,
+	                             int32_t size, int32_t);
+	static void CreateTownSpell(int32_t townId, const std::string& spellName);
+	static void CreateNewTownSpell(int32_t townId, const std::string& spellName);
+	static void CreateTownCentreSpellIcon(int32_t, const std::string&);
+	static void CreateSpellIcon(glm::vec3 position, const std::string&, int32_t, int32_t, int32_t);
+	static void CreatePlannedSpellIcon(int32_t, glm::vec3 position, const std::string&, int32_t, int32_t, int32_t);
+	static void CreateVillager(glm::vec3, glm::vec3, const std::string&);
+	static void CreateTownVillager(int32_t, glm::vec3, glm::vec3, int32_t);
```

Comment:
Getting a runtime error with the current function arguments. This fixes it.

```suggestion
	static void CreateTownVillager(int32_t townId, glm::vec3 position, const std::string& villagerType, int32_t age);
```